graph-query-ir/src/main/java/oracle/pgql/lang/PreparedStatement.java

```diff
@@ -136,6 +138,16 @@
   void setTimestampWithTimezone(int parameterIndex, OffsetDateTime x) throws PgqlException;
 
   /**
+   * Sets the designated parameter to the given Point2D value.
+   *
+   * @param parameterIndex
+   *          the first parameter is 1, the second is 2, ...
+   * @param x
+   *          the parameter value
+   */
+  void setPoint2D(int parameterIndex, Point2D x) throws PgqlException;
+
+  /**
```

Comment:
Hi Wilco,

It may be nice to separate the Geo-related functionality out so that other implementations of PGQL can choose to support it or not.

Perhaps we can put everything into a `spatial` package:

`oracle.pgql.lang.spatial.Point`
`oracle.pgql.lang.spatial.STPreparedStatement`
`oracle.pgql.lang.spatial.STResultSet` (in the future)